### ISSUE TITLE
Move events to Glean

### ISSUE THREAD

**Issue by eliserichards** [mozilla-lockwise/lockwise-android#1111]
Related to #127

**Comment by jhugman** [comment]
This will require Data Review, and is a candidate for pairing with @Dexterp37

**Comment by Dexterp37** [comment]
> 
> 
> This will require Data Review, and is a candidate for pairing with @Dexterp37

Yup. Note that we currently completed the [enumeration of events](https://docs.google.com/spreadsheets/d/1qR_IozmnVBCkml9-P6QNNsVZ5rWiDZXRHkVIFhDPtH0/edit#gid=0) and are waiting for information by David. Will reach out once we know more.
